source/GameData.cpp

```diff
@@ -808,6 +808,15 @@ double GameData::SolarWind(const Sprite *sprite)
 
 
 
+// Get the map icon of the given stellar object sprite.
+const Sprite *GameData::StarIcon(const Sprite *sprite)
+{
+	auto it = objects.starIcons.find(sprite);
```

Comment:
```suggestion
	const auto it = objects.starIcons.find(sprite);
```
I think this can be const?